src/ucp/api/ucp.h

```diff
@@ -631,6 +633,62 @@ void ucp_worker_progress(ucp_worker_h worker);
 
 
 /**
+ * @ingroup UCP_WORKER
+ * @brief Obtain an event file descriptor for event notification.
+ *
+ * This routine returns a valid file descriptor for polling functions.
+ * The file descriptor   will get signaled when an event occurs.
```

Comment:
This is a high level API and we have to be more descriptive about the API.
Please describe what is the "event" and what is the "signal". Two lines of documentation is not enough.
Also describe how the fd can be used and what is the goal of this functionality. What can generate the "event" ?